### PR TITLE
refactor(disconnect): OAuth services Disconnect button does not need …

### PR DESCRIPTION
**Pull request by divyabiyani**
Fix for #4292 .

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
@ryanfeeley - The same button is used for both OAuth clients and Sync connected devices. Should the `...` be removed from both?

**Comment by ryanfeeley** [comment]
@shane-tomlinson No, because the … lets the user know that the action is revocable. With Sync it is, with OAuth it is not (yet).

**Comment by vladikoff** [comment]
This PR needs a small update @divyabiyani 

we need a conditional based on `{{clientType}}` , if `clientType` is `oAuthApp` then we **should not** show the `…` OR if the `clientType` is  `device` then show the `…`, whatever is easier.

**Comment by vladikoff** [comment]
@divyabiyani I saw your IRC message, your version of the `content-server` seems out of date. Make sure to either `git pull origin master` to get latest changes on the `master` branch OR `git rebase origin/master` to rebase your new fix branch against master.

**Comment by divyabiyani** [comment]
@vladikoff Yes now it works.
Thankyou!

**Comment by divyabiyani** [comment]
@vladikoff My CircleCi check is failing after the tunnel is started.
Could you please help me with it?

File: app/scripts/models/device.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ define(function (require, exports, module) {
       clientType: Constants.CLIENT_TYPE_DEVICE,
       id: null,
       isCurrentDevice: null,
+      isDevice: true,
```
Comment (vladikoff) [inline app/scripts/models/device.js]: 👍

File: app/scripts/templates/settings/clients.mustache [hunk 2]
```diff
@@ -26,8 +26,7 @@
             {{^isCurrentDevice}}
             <div class="last-connected">{{lastAccessTimeFormatted}}</div>
             {{/isCurrentDevice}}
-            {{#isDevice}}<button class="settings-button warning client-disconnect" data-id="{{id}}" data-type="{{clientType}}">{{#t}}Disconnect…{{/t}}</button>{{/isDevice}}
-            {{^isDevice}}<button class="settings-button warning client-disconnect" data-id="{{id}}" data-type="{{clientType}}">{{#t}}Disconnect{{/t}}</button>{{/isDevice}}
+            <button class="settings-button warning client-disconnect" data-id="{{id}}" data-type="{{clientType}}">{{#isDevice}}{{#t}}Disconnect…{{/t}}{{/isDevice}}{{^isDevice}}{{#t}}Disconnect{{/t}}{{/isDevice}}</button>
```
Comment (vladikoff) [inline app/scripts/templates/settings/clients.mustache]: @divyabiyani this line seems way too long now, we should probably line break before `{{#isDevice}}` and clean it up.

File: app/scripts/templates/settings/clients.mustache [hunk 3]
```diff
@@ -26,7 +26,8 @@
             {{^isCurrentDevice}}
             <div class="last-connected">{{lastAccessTimeFormatted}}</div>
             {{/isCurrentDevice}}
-            <button class="settings-button warning client-disconnect" data-id="{{id}}" data-type="{{clientType}}">{{#t}}Disconnect…{{/t}}</button>
+            <button class="settings-button warning client-disconnect" data-id="{{id}}" data-type="{{clientType}}">
+            {{#isDevice}}{{#t}}Disconnect…{{/t}}{{/isDevice}}{{^isDevice}}{{#t}}Disconnect{{/t}}{{/isDevice}}</button>
```
Comment (vladikoff) [inline app/scripts/templates/settings/clients.mustache]: please see the style above in this template:

```html
             {{#isCurrentDevice}}
             <div class="device-current">{{#t}}current device{{/t}}</div>
             {{/isCurrentDevice}}
              {{^isCurrentDevice}}
              <div class="last-connected">{{lastAccessTimeFormatted}}</div>
              {{/isCurrentDevice}}
```

Let's apply the same to `isDevice` so it looks cleaner

Comment (vladikoff) [inline app/scripts/templates/settings/clients.mustache]: @divyabiyani you can indent the two `{{#t}}Disconnect` by 2 spaces to the right. Thanks for making the updates!

Comment (divyabiyani) [inline app/scripts/templates/settings/clients.mustache]: Ok @vladikoff I will do these changes.
Thanks for guiding me.

**Comment by vladikoff** [comment]
image

Nice!

**Pull request merged**